### ISSUE TITLE
Scaffolding date/datetime properties

### ISSUE THREAD

**Issue by TimGeyssens** [TimGeyssens/UIOMatic#56]
Seems these get a min date value but then can't save object with those values

```
An error occured

Could not convert string to DateTime: 0000-12-31 22:00:00. Path 'Birthday', line 1, position 40.

EXCEPTION DETAILS

Newtonsoft.Json.JsonReaderException: Could not convert string to DateTime: 0000-12-31 22:00:00. Path 'Birthday', line 1, position 40.
```

**Comment by TimGeyssens** [comment]
Reverted the changes since the date(time) I was checking on depended on my timezone

**Comment by TimGeyssens** [comment]
So when scaffolding it should just get an empty value

**Comment by mattbrailsford** [comment]
I'll address this in the other issue, but it's probably get DateTime.MinValue. I was thinking about this, and it'll work with DateTime? (nullable) having an empty value, but I think it should be possibele / allowed that your model says DateTime (requiring a value) and the UI should provide it, so we do need a way of handling "un set" values, but like I say, I'll handle that.

**Comment by TimGeyssens** [comment]
Okido, awesome, looking very good for the rest, hard to find bugs any bugs 👍

**Comment by mattbrailsford** [comment]
Closing as will get fixed as part of issue #59
